src/Util/OpenGamePrompt.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+
+namespace sodoff.Utils;
+
+public class OpenGamePrompt {
+    public static void prompt() {
+        string key;
+        Console.WriteLine("Do you want to open the game");
+        Console.WriteLine("yes or no");
+        key = Console.ReadLine();
+        if (key == "yes")

```

Comment:
This block seems to be poor code practice.  This `if` will match _only_ `yes`, not `Yes`, `y`, `Y`, or anything else, and the following `if` will match _only_ `no`, not `No`, `n`, `N`, or anything else.  Also, if neither `if` matches, it'll just print nothing and silently carry on.   I would change `key` in both this `if` and the following one to `key.First().ToLower()`, which would match `yes`, `Yes`, `y`, `Y`, or anything that starts with an upper- or lowercase Y in this block, and `no`, `No`, `n`, `N`, or anything starting with upper- or lowercase N in the following block.  Or at very least, `key.ToLower()`

Reply:
i have added that i used key.StartsWith
i added else statement

Reply:
and is it ok if i add a icon  for the server executable?
![7ej8x79p](https://github.com/user-attachments/assets/fde2e9aa-051e-402a-812b-2caa606e29d2)
i have this icon